README.md

```diff
@@ -1 +1,3 @@
-Hello World!
+# Arrow Flight Client Examples
+
+This repository provides sample Arrow Flight clients in serveral languages to help you get started with Arrow Flight with Dremio.
```

Comment:
Is this a repo of examples showing you how to use the clients, or a repo of the clients themselves? Seems like the former, but the docs seem to suggest the latter?

Reply:
Changed wording, please check.